src/frontend/magnify/src/types/group.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { Member } from './member.interface';
```

Comment:
Adding `interface` in the file name is useless IMHO. You are in a `types` directory, it's already explicit.